app/Http/Controllers/VideoController.php

```diff
@@ -187,4 +188,26 @@ public function deleteVideo(Request $request)
         }
     }
 
+    // Like Module
+    public function toggleLikeDislike(VideoLikeRequest $request)
+    {
+        try {
+            $video_id = $request->video_id;
+            
+            if($this->currentUser->likeVideos()->where('video_id', $video_id)->exists()) 
+            {
+                $this->currentUser->likeVideos()->detach($video_id);
+                return new BaseResponse(STATUS_CODE_OK, STATUS_CODE_OK, "Video Unliked.");
+            } else {
+                $this->currentUser->likeVideos()->attach($video_id, ['rating' => $request->rating]);
+                
+                // Notification needed for other user.
```

Comment:
Should implement notification module as soon as possible for the remaining modules.